cluster/images/hyperkube/Makefile

```diff
@@ -18,20 +18,42 @@
 #   [ARCH=amd64] [REGISTRY="staging-k8s.gcr.io"] make (build|push) VERSION={some_released_version_of_kubernetes}
 
 REGISTRY?=staging-k8s.gcr.io
+IMAGE ?= $(REGISTRY)/hyperkube
```

Comment:
I don't think you need to update the hyperkube Makefile; humans don't directly call `make` in here; instead, it's called through the build scripts `build/lib/release.sh`, and then your work in https://github.com/kubernetes/release/pull/622 ensures a manifest list is created for hyperkube.